hack/errata.py

```diff
@@ -90,6 +92,58 @@ def notify(message, webhook=None):
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    pulls = repo.get_pulls(state='open', sort='created', base='master')
+    for pr in pulls:
+        try:
+            # Skip unknown PRs
+            if not pr.title.startswith("Enable "):
+                continue
+            # Ignore PRs which don't target fast
+            if pr.title.split(" ")[3] != "fast":
+                continue
+            yield pr.number, pr.body
+        except Exception as e:
+            print(f"Failed to parse {pr.number}: {e}")
+
+
+def extract_errata_number_from_body(body):
+    ERRATA_MARKER = 'https://errata.devel.redhat.com/advisory/'
+    first_line = body.split('\n')[0]
+    links = [
+        x for x in first_line.split(' ') if x.startswith(ERRATA_MARKER)
+    ]
+    if len(links) == 0:
+        return None
+    errata_num = links[0].split('/')[-1]
+
+    try:
+        return int(errata_num)
+    except ValueError:
+        return None
+
+
+def lgtm_fast_pr_for_errata(message):
+    github_token = os.environ.get("GITHUB_TOKEN")
+    if not github_token:
+        print(f"Skipping fast PR check: no github token set")
+        return
+
+    github_object = Github(github_token)
+    repo = github_object.get_repo("openshift/cincinnati-graph-data")
```

Comment:
Nit: Add argparse options for this repo and the token, so they are less magical and more of an explicit config.